trimmer.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class TrimmerDecorator < Basedecorator
+  def correct_name
+    return super.slice(0..9) unless super.length <= 10
+  end
```

Comment:
**OPTIONAL** - According to the project requirements, this method should trim the name if it is longer than 10 chars or return the name if it is less than 10 chars. Please include a condition to return the name when it is less than 10 chars

Reply:
Let's put optional ones in our pocket